src/components/pageLayout/pageHeader.js

```diff
@@ -10,26 +10,32 @@ import {
  *
  * @param {object} props
  * @param {Node} props.children
+ * @param {string} props.title
  * @returns {Node}
  */
-const PageHeader = ({ children }) => (
+const PageHeader = ({ children, title }) => (
```

Comment:
If I make the heading and subheading both children of the PageHeader, there doesn't seem to be much of any reason to even keep our custom PageHeader element.  We could just use the "RcsPageHeader" directly in the views and delete the custom one.  (I assume we wanted this "sub heading" in the header element.)

Reply:
Also, I made the title not required because we do have a situation in which we don't need the title, specifically, the Loader.  The loader adds a graphical component into a title component when it could just be loading it directly into the PageHeader.

Reply:
This falls under one of the learned behaviors on this project's history, and those multiple React apps that have come before this one when consuming components from resources not controlled by us. 

You may see this as a simplification, but given components are prone to change at a seemingly random occurrence we've created a predictable outcome. Not a fan of having to scramble when something melts, multiple React setups have taught us this. We have no plans to change this thinking. 

We can continue this conversation off to the side.